accepted/2020/or-greater-defines/or-greater-defines.md

```diff
@@ -0,0 +1,221 @@
+# OR_GREATER preprocessor symbols for TFMs
```

Comment:
Was there a discussion around the name, e.g. I'd have intuitively called this `OR_LATER` instead.

Reply:
I proposed OR_LATER before and I was told that OR_GREATER is more consistent with other existing prior art.